com.kms.cura.dal/src/com/kms/cura/dal/database/DoctorUserDatabaseHelper.java

```diff
@@ -154,6 +150,30 @@ protected void insertToDoctorReferenceTables(DoctorUserEntity entity) throws SQL
 		insertReferenceRowsToReferenceTable(Doctor_SpecialityColumn.TABLE_NAME, referenceRows);
 	}
 
+	protected void insertWorkingHour(DoctorUserEntity doctorUserEntity) throws SQLException {
+		List<OpeningHour> hours = doctorUserEntity.getFacility().get(0).getOpeningHours();
+		PreparedStatement stmt = null;
+		StringBuilder builder = new StringBuilder();
+
+		builder.append("INSERT INTO ");
+		builder.append(Doctor_FacilityColumn.TABLE_NAME);
+		builder.append(" VALUES (?, ?, ?, ?, ?);");
+		try{
+			for(int i=0; i<hours.size();++i){
+				stmt = con.prepareStatement(builder.toString());
+				stmt.setInt(1,Integer.parseInt(doctorUserEntity.getId()));
+				stmt.setInt(2,Integer.parseInt(doctorUserEntity.getFacility().get(0).getId()));
```

Comment:
Extract doctorUserEntity.getId() and doctorUserEntity.getFacility().get(0) for re-use with this loop